src/styles/clock.scss

```diff
@@ -0,0 +1,29 @@
+html {
+  background: #282828;
+  text-align: center;
+  font-size: 10px;
+}
+
+body {
+  margin: 0;
+  font-size: 2rem;
+  display: flex;
+  flex: 1;
+  min-height: 100vh;
+  align-items: center;
+}
```

Comment:
1. move to reset.scss
2. there are a lot styles for body, leave only margin, anf move others into separated container 
3. smae for html,` text-align: center;` looks like doesn't belong there